libs/transactions/src/lib/extensions/transaction-addition-details.ts

```diff
@@ -0,0 +1,13 @@
+import {InjectionToken, Type} from "@angular/core";
+import { TransactionItem } from "@backbase/data-ang/transactions";
+import {ExtensionComponent} from "../extension-slot.directive";
+
+export type TransactionAdditionalDetailsComponent = ExtensionComponent<TransactionItem>;
```

Comment:
As an example of a best practice, I think we shouldn't use the entire `TransactionItem` as the component's context, unnecessarily exposing the entire object as (basically) part of the public API

Reply:
Yeah, that's easily dealt with.  Will refactor.